cmd/bosun/web/static/partials/config.html

```diff
@@ -33,18 +32,90 @@
 	color: lightgray;
 }
 .lines .lineerror {
-	color: #a94442;
-	background: #f2dede;
+	color: black;
+	font-weight: bold;
+	background-color: lightgray;
 }
+
 </style>
+<label style="float:left; margin-right:15px;">Jump to:</label>
+<div class='row'>
+<div class="dropdown" ng-repeat="(type,list) in items" style="float:left; margin-right:5px;margin-bottom:15px;">
```

Comment:
lotta inline styles. since there are so many, and ofter similar, let's make some classes instead.